.github/workflows/main.yml

```diff
@@ -80,3 +80,20 @@ jobs:
         working-directory: ${{ env.GOPATH }}/src/k8s.io/kops
         run: |
           make quick-ci
+
+  tests-e2e-scenarios-bare-metal:
```

Comment:
Should we move the test to a different file?

Reply:
I was debating it ... yes, I think it's a good idea!  `presubmits.yaml` ?  `e2e.yaml` ?

Reply:
`e2e` or `metal`. We can always rename later.

Reply:
e2e works for me - thanks!